packages/fxa-payments-server/src/components/NewUserEmailForm/index.tsx

```diff
@@ -78,22 +78,30 @@ export const NewUserEmailForm = ({
     <Form
       data-testid="new-user-email-form"
       validator={validator}
-      className="new-user-email-form"
+      className="new-user-email-form relative"
       {...{ onChange }}
     >
       <Localized
         id="new-user-sign-in-link"
         elems={{
           a: (
-            <a onClick={onClickSignInButton} href={signInURL}>
+            <a
```

Comment:
Hovering over this element seems to behave differently on Stage vs Locally. (Locally it's showing as blue on hover)

Reply:
Fixed! Modifier added. Thanks for the catch!

Note to self for future: when I removed the style for `a`, I had actually removed the style for `.new-user-email-form .sign-in-copy a`, the specificity that was overriding default hover state styles.